pkg/action/build-uki.go

```diff
@@ -148,7 +150,7 @@ func (b *BuildUKIAction) Run() error {
 		}
 		// Then build the image
 
-		err = b.createContainer(b.outputDir, kairosVersion)
+		err = b.createContainer(b.outputDir, kairosVersion, b.name)
```

Comment:
`b` is available in `createContainer` the same way it happens in `createISO` above. Why pass it as an argument here?

Reply:
Ah, good catch. I oriented myself on the other parameters (`b.outputDir, kairosVersion`).
I'll update the PR.

Reply:
Updated